CHANGELOG.txt

```diff
@@ -21,6 +21,7 @@ master:
      resource_ids which are not found via the normal means (see #2279).
    * Calling Stream.write(...) on an empty stream will now raise an
      ObsPyException consistently across all I/O plugins (see #2201)
+   * Stream.get_gaps() will now check for gaps within previously merged Traces.
```

Comment:
I think this should be rephrased a bit to make it clear this is about masked-array Traces, i.e. merged with `Stream.merge(0)`, because obviously `get_gaps()` is not able to know about gaps when gaps are filled with whatever values.

Reply:
That makes sense. I will make the changes you suggest here and in the next comment.